### PR TITLE
Handled Summary experiences

### PR DESCRIPTION
**Pull request by HeahDude**

### CODE REVIEW

File: src/Entity/Adherent.php [hunk 1]
```diff
@@ -158,13 +158,6 @@ class Adherent implements UserInterface, GeoPointInterface, EncoderAwareInterfac
      */
     private $memberships;
 
-    /**
-     * @var Summary|null
-     *
-     * @ORM\OneToOne(targetEntity="Summary", mappedBy="member", cascade={"all"})
-     */
-    private $summary;
```
Comment (HeahDude) [inline src/Entity/Adherent.php]: @tgalopin This is lazy :).

File: src/Summary/SummaryManager.php [hunk 2]
```diff
@@ -25,4 +31,33 @@ public function getForAdherent(Adherent $adherent): Summary
...
+        $this->updateSummary($summary);
+    }
+
+    public function removeExperience(Adherent $adherent, JobExperience $experience): bool
+    {
+        if (!$summary = $this->repository->findOneForAdherent($adherent)) {
+            return false;
+        }
+
+        SummaryItemDisplayOrderer::removeItem($summary->getExperiences(), $experience);
```
Comment (HeahDude) [inline src/Summary/SummaryManager.php]: Hmm I forgot to move that call in the manager :p.

File: src/Form/SummaryItemPositionType.php [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+        $builder->addEventListener(FormEvents::SUBMIT, function (FormEvent $event) use ($item, $collection) {
+            $form = $event->getForm();
+            $oldPosition = $form->getData();
+            $newPosition = $form->get('entry')->getData() ?: 1;
+
+            if ($collection) {
+                if ($item && $oldPosition !== $newPosition) {
+                    SummaryItemDisplayOrderer::updateItem($collection, $item, $oldPosition, $newPosition);
+                } else {
```
Comment (HeahDude) [inline src/Form/SummaryItemPositionType.php]: This should be `elseif(!$item)`.

File: templates/summary/_block/languages.html.twig [hunk 4]
```diff
@@ -1,11 +1,12 @@
 <section class="l__wrapper--medium">
     {% block summary_languages %}
-        <div id="summary_languages" class="l__row l__row--h-stretch l__row--top l__tablet--col b__nudge--bottom-large">
+        <div class="l__row l__row--h-stretch l__row--top l__tablet--col b__nudge--bottom-large">
```
Comment (illusionOfParadise) [inline templates/summary/_block/languages.html.twig]: Why did you remove `id` ?

Comment (illusionOfParadise) [inline templates/summary/_block/languages.html.twig]: Ok, you added a class
